web3/utils/__init__.py

```diff
@@ -0,0 +1,28 @@
+from web3._utils import (  # noqa: F401
+    abi,
+    blocks,
+    caching,
+    compat,
+    contracts,
+    datatypes,
+    decorators,
+    empty,
+    encoding,
+    ens,
+    events,
+    filters,
+    formatters,
+    function_identifiers,
+    http,
+    hypothesis,
+    math,
+    module_testing,
+    normalizers,
+    request,
+    rpc_abi,
+    six,
+    threads,
+    toolz,
+    transactions,
+    validation,
+)
```

Comment:
@carver @pipermerriam Is this the right way to alias `utils` -> `_utils`?

Reply:
I don't have a strong opinion on it. Seems okay to me.